compute/managed-instances/demo/main.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python3
```

Comment:
We should add the copyright boilerplate here; copy from any another file.

Reply:
Done